src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -161,6 +162,20 @@ def load_command_table(self, _):
     with self.command_group('sql elastic-pool', database_operations) as g:
         g.command('list-dbs', 'list_by_elastic_pool')
 
+    ###############################################
+    #             sql failover-group              #
+    ###############################################
+    failover_groups_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations.failover_groups_operations#FailoverGroupsOperations.{}',
+        client_factory=get_sql_failover_groups_operations)
+    with self.command_group('sql failover-group', failover_groups_operations, client_factory=get_sql_failover_groups_operations) as g:
+        g.command('show', 'get')
+        g.command('list', 'list_by_server')
+        g.custom_command('create', 'failover_group_create')
+        g.custom_command('update', 'failover_group_update')
```

Comment:
Why custom_command instead of generic_update_command?

Reply:
Note that custom function for generic_updated_command does not work the same as custom_command